plugins/node/opentelemetry-instrumentation-aws-sdk/src/types.ts

```diff
@@ -62,6 +64,10 @@ export interface AwsSdkSqsProcessCustomAttributeFunction {
   (span: Span, sqsProcessInfo: AwsSdkSqsProcessHookInformation): void;
 }
 
+export type AwsSdkDynamoDBStatementSerializer = (
+  commandInput: CommandInput
+) => string | undefined;
```

Comment:
I wonder if the serializer should also get the `operation` to be able to properly sanitize. I guess the sanitization logic can probably work differently for different operations? 

This can also be added in the future if someone brings up the need.

Reply:
I like the suggestion. As pointed out elsewhere, the sanitize function is supplied as part of the public API so I think including `operation` now just avoids breaking changes later 👍🏻